portfolio/src/main/webapp/style.css

```
@@ -1,8 +1,28 @@
-/* Styling for comment delete button */
-.delete {
+/* Style for div holding sorting options for comments */
+.sort_div {
+    padding-top: 10px;
+}
+
+/* style the select structure */
```

```suggestion
/* Style the select structure */
```

That said, I'm not sure the comments in this file really add too much. Ideally the class/id names are useful enough to not require comments.

---

should I take them out or can I leave them for high level clarification?

---

Up to you - i'd say omit them.